Dockerfile

```diff
@@ -1,11 +1,12 @@
-FROM --platform=$BUILDPLATFORM golang:1.19.2-alpine AS build
+FROM --platform=$BUILDPLATFORM golang:1.19.2-bullseye AS build
```

Comment:
Switching from Alpine to Ubuntu doesn't increase image size, right?  (I know it used to, I know it should've been fixed many years ago, but asking for old times' sake.)